openedx/features/course_experience/api/v1/serializers.py

```diff
@@ -0,0 +1,12 @@
+"""
+Serializer for Course Deadlines (Mobile)
+"""
+from rest_framework import serializers
+from lms.djangoapps.course_home_api.mixins import DatesBannerSerializerMixin
+
+
+class CourseDeadlinesMobileSerializer(DatesBannerSerializerMixin):
+    content_type_gating_enabled = serializers.BooleanField()
+    missed_deadlines = serializers.BooleanField()
+    missed_gated_content = serializers.BooleanField()
+    verified_upgrade_link = serializers.URLField()
```

Comment:
Is it necessary to do this as opposed to just saying the serializer for your view should just be the `DatesBannerSerializerMixin`? That is what actually contains all your data.

Reply:
i would say just keep this Serializer, but just put a `pass`, since it already inherits from the serializer.  That way it can still act as a placeholder if mobile needs any custom additional fields which arent already in the mixin.